app/assets/stylesheets/components/form/_form.scss

```diff
@@ -5,3 +5,18 @@
 .form__input-container--medium {
   max-width: 140px;
 }
+
+%form-row-2up {
```

Comment:
I just put the angle brackets on. @spencer?

Might be worth making the wireframes visible to you in future. Would that be useful?